### ISSUE TITLE
Error when using UglifyJS

### ISSUE THREAD

**Issue by chafreaky** [weifeiyue/vue-datepicker-local#8]
Hey there!

Thank you for this awesome datepicker package, loving its simplicity :)

When we try to bundle it into our production build, using UglifyJS, we get this as an error
`Unexpected token: operator (>) [./~/vue-datepicker-local/src/index.js:2,0]`

Seems related to the es6 syntax in `index.js` below:

```
const install = Vue => {
  Vue.component(component.name, component)
}
```

Any idea how to fix it? 
It happened to other packages as well ([link](https://github.com/ankurk91/vue-flatpickr-component/issues/3)) but I don't want to rewrite the syntax just to fix this... 

Thanks again!

**Comment by weifeiyue** [comment]
@chafreaky What's your babel configuration?

**Comment by weifeiyue** [comment]
@chafreaky Or you can refer to my configuration

**Comment by chafreaky** [comment]
It's the vanilla one from `vue-cli`, see below:

```
var path = require('path')
var fs = require('fs')
var utils = require('./utils')
var config = require('../config')
var vueLoaderConfig = require('./vue-loader.conf')

function resolve (dir) {
  return path.join(__dirname, '..', dir)
}

module.exports = {
  entry: {
    app: './src/main.js'
  },
  output: {
    path: config.build.assetsRoot,
    filename: '[name].js',
    publicPath: process.env.NODE_ENV === 'production'
      ? config.build.assetsPublicPath
      : config.dev.assetsPublicPath
  },
  resolve: {
    extensions: ['.js', '.vue', '.json'],
    alias: {
      'vue$': 'vue/dist/vue.esm.js',
      '@': resolve('src'),
    },
    symlinks: false
  },
  module: {
    rules: [
      {
        test: /\.(js|vue)$/,
        loader: 'eslint-loader',
        enforce: 'pre',
        include: [resolve('src'), resolve('test')],
        options: {
          formatter: require('eslint-friendly-formatter')
        }
      },
      {
        test: /\.vue$/,
        loader: 'vue-loader',
        options: vueLoaderConfig
      },
      {
          test: /\.css$/,
          loader: 'css-loader',
      },
      {
        test: /\.js$/,
        loader: 'babel-loader',
        include: [resolve('src'), resolve('test')],
      },
      {
        test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
        loader: 'url-loader',
        options: {
          limit: 10000,
          name: utils.assetsPath('img/[name].[hash:7].[ext]')
        }
      },
      {
        test: /\.(mp4|webm|ogg|mp3|wav|flac|aac)(\?.*)?$/,
        loader: 'url-loader',
        options: {
          limit: 10000,
          name: utils.assetsPath('media/[name].[hash:7].[ext]')
        }
      },
      {
        test: /\.(woff2?|eot|ttf|otf)(\?.*)?$/,
        loader: 'url-loader',
        options: {
          limit: 10000,
          name: utils.assetsPath('fonts/[name].[hash:7].[ext]')
        }
      },
    ]
  }
}
```

**Comment by weifeiyue** [comment]
@chafreaky you can change babel-loader config like this
```js
{
  test: /\.js$/,
  loader: 'babel-loader'
  //include: [resolve('src'), resolve('test')],
}
```
and change .babelrc file like this
```json
{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": ["last 2 versions"]
      }
    }]
  ]
}
```
after that，it will work~

**Comment by vikramkute** [comment]
@chafreaky I can confirm this.

@weifeiyue I am getting same error only while bundling for production. This is because UglifyJs does not support ES6+ and Vue Cli use UglifyJs in webpack template. It throws error but does not break anything.

This is common error for those who are using vue-cli webpack (UglifyJs)
This error could go away if you could change arrow function syntax to regular function in index.js.

```
import component from './VueDatepickerLocal.vue'

const vueDatepickerLocalPlugin = {
  install: function(Vue) {
    Vue.component(component.name, component)
  }
}
component.install = vueDatepickerLocalPlugin.install
if (typeof window !== 'undefined' && window.Vue) {
  vueDatepickerLocalPlugin.install(window.Vue)
}
export default component
```

**Comment by weifeiyue** [comment]
@chafreaky   fixed [@v1.0.8](https://github.com/weifeiyue/vue-datepicker-local/releases/tag/v1.0.8)

**Comment by chafreaky** [comment]
Thank you @weifeiyue & @vikramkute!
